elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -2,9 +2,13 @@
 
 const TABWIDTH = 50;      // defines how many px the answer block is indented by, when the student
                           // drags and indents a block
-const INDENT_OFFSET = 10;  // For aesthetic, all answer blocks are offseted to the right by
+const INDENT_OFFSET = 0;  // For aesthetic, all answer blocks are offseted to the right by
                           // 5px, so the answer tiles are not directly touching the dropzone margins
```

Comment:
This comment seems to not match what the code is actually doing anymore. (as an aside, this is one of the reasons I dislike comments which simply restate what the code is doing)

Reply:
Yes, the offset was made obsolete after my refactoring (which was done for a code interview :laughing: ), this is why the comment wasn't updated... I'll be updating the code shortly